files/egjs/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "NAVER corp"
+github = "https://github.com/naver/egjs"
+homepage = "http://naver.github.io/egjs"
+description = "Set of UI interactions, effects and utilities components library."
+mainfile = "dist/eg.js"
```

Comment:
I don't understand your intention
Our main files are eg.js and eg.min.js
What should I do?